app/src/main/java/edu/uc/coffeens/stocktracker/ui/main/MainViewModel.kt

```diff
@@ -31,15 +31,18 @@ class MainViewModel : ViewModel() {
         stock = stockService.fetchStock()
     }
 
-    fun save(stocks: Stock) {
+    /**
+     * TODO: Implement data save function
+     */
+    fun save(stock: Stock) {
```

Comment:
Changed function parameter name to match the passing variable for clarity https://kotlinlang.org/docs/functions.html#parameters

Reply:
If adding KDoc, complete it with proper documentation.  Adding it with a TODO isn't adding much value.